docs/networking/kubeovn-vpc.md

```diff
@@ -0,0 +1,667 @@
+##  **Virtual Private Cloud (VPC) - Concepts & Architecture** 
+
+***Overview*** 
+
+A **Virtual Private Cloud (VPC)** is a logically isolated network that provides full control over IP addressing, subnets, route tables, firewalls, and gateways within a cloud infrastructure. VPC allows the secure and scalable deployment of virtualized resources such as compute, storage, and container services.
+
+The following diagram shows a typical VPC architecture with both public and private subnets to separate internet-facing traffic from internal resources.
+
+
+###  VPC → Subnet → Overlay Network → VM: Hierarchical Diagram
+
+#### Component Relationship Table
+
+| Layer | Component         | Example in Harvester        | Description                                                                 |
+|-------|-------------------|-----------------------------|-----------------------------------------------------------------------------|
+| L3    | VPC               | `vpc-1`                     | Logical network container managing multiple subnets and routing/NAT rules. |
+| L3    | Subnet            | `vswitch1-subnet`           | IP segment (CIDR); each subnet is bound to one unique Overlay Network.     |
+| L2    | Overlay Network   | `vswitch1`                  | Virtual Layer 2 switch that connects VMs; carries subnet traffic.          |
+| L2/L3 | Virtual Machine   | `vm1-vswitch1`              | Attached to an Overlay Network; receives IP/Gateway from its subnet.       |
+
+#### ASCII Diagram
```

Comment:
nit: Use a diagram drawing instead of ASCII for consistency with other pages. May need @jillian-maroket help here in the future.